src/std/misc/list.ss

```diff
@@ -0,0 +1,92 @@
+;; -*- Gerbil -*-
+package: std/misc
+;;;; List utilities
+
+(export
+  plist->alist
+  length=? length=n?
+  length<? length<n? length<=? length<=n?
+  length>? length>n? length>=? length>=n?
+  call-with-list-builder
+  snoc append1)
+
+(import
+  :std/error :std/srfi/1)
+
+;; This function transform a property list (k1 v1 k2 v2 ...) into
+;; an association list ((k1 . v1) (k2 . v2) ...).
+;; NB: it uses equal? as the equality predicate for keys.
```

Comment:
this comment is redundant and misleading.

Reply:
I assume you mean just the last line. Removed.